agent/azure-monitor-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/AzureMonitorTraceExporter.java

```diff
@@ -73,132 +42,33 @@ public final class AzureMonitorTraceExporter implements SpanExporter {
 
   private static final ClientLogger LOGGER = new ClientLogger(AzureMonitorTraceExporter.class);
 
-  private static final Set<String> SQL_DB_SYSTEMS;
-
-  private static final Trie<Boolean> STANDARD_ATTRIBUTE_PREFIX_TRIE;
-
-  // TODO (trask) add to generated ContextTagKeys class
-  private static final ContextTagKeys AI_DEVICE_OS = ContextTagKeys.fromString("ai.device.os");
-
-  // TODO (trask) this can go away once new indexer is rolled out to gov clouds
-  private static final AttributeKey<List<String>> AI_REQUEST_CONTEXT_KEY =
-      AttributeKey.stringArrayKey("http.response.header.request_context");
-
-  public static final AttributeKey<String> AI_OPERATION_NAME_KEY =
-      AttributeKey.stringKey("applicationinsights.internal.operation_name");
-  public static final AttributeKey<String> AI_LEGACY_PARENT_ID_KEY =
-      AttributeKey.stringKey("applicationinsights.internal.legacy_parent_id");
-  public static final AttributeKey<String> AI_LEGACY_ROOT_ID_KEY =
-      AttributeKey.stringKey("applicationinsights.internal.legacy_root_id");
-
-  // this is only used by the 2.x web interop bridge
-  // for ThreadContext.getRequestTelemetryContext().getRequestTelemetry().setSource()
-  private static final AttributeKey<String> AI_SPAN_SOURCE_KEY =
-      AttributeKey.stringKey("applicationinsights.internal.source");
-  private static final AttributeKey<String> AI_SESSION_ID_KEY =
-      AttributeKey.stringKey("applicationinsights.internal.session_id");
-  private static final AttributeKey<String> AI_DEVICE_OS_KEY =
-      AttributeKey.stringKey("applicationinsights.internal.operating_system");
-  private static final AttributeKey<String> AI_DEVICE_OS_VERSION_KEY =
-      AttributeKey.stringKey("applicationinsights.internal.operating_system_version");
-
-  private static final AttributeKey<String> AZURE_NAMESPACE =
-      AttributeKey.stringKey("az.namespace");
-  private static final AttributeKey<String> AZURE_SDK_PEER_ADDRESS =
-      AttributeKey.stringKey("peer.address");
-  private static final AttributeKey<String> AZURE_SDK_MESSAGE_BUS_DESTINATION =
-      AttributeKey.stringKey("message_bus.destination");
-  private static final AttributeKey<Long> AZURE_SDK_ENQUEUED_TIME =
-      AttributeKey.longKey("x-opt-enqueued-time");
-
-  private static final AttributeKey<Long> KAFKA_RECORD_QUEUE_TIME_MS =
-      AttributeKey.longKey("kafka.record.queue_time_ms");
-  private static final AttributeKey<Long> KAFKA_OFFSET = AttributeKey.longKey("kafka.offset");
-
   private static final OperationLogger exportingSpanLogger =
-      new OperationLogger(AzureMonitorTraceExporter.class, "Exporting span");
+      new OperationLogger(SpanDataMapper.class, "Exporting span");
```

Comment:
should this get reverted? export is done by the trace exporter. spandatamapper is doing the mapping only.

Reply:
reverted 👍